presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -816,6 +928,150 @@ private static boolean testValue(Type type, Object value, TupleDomainFilter filt
         return false;
     }
 
+    private interface SubfieldPruner
+    {
+        Object prune(Object value);
```

Comment:
I don't have strong opinion on using `Object`. Ideally, it could be `Prunable` or some interfaces. But ok to keep as is given it's a test.